core/domain/story_domain_test.py

```diff
@@ -837,154 +744,3 @@ def test_story_contents_export_import(self):
             story_contents_dict)
         self.assertEqual(
             story_contents_from_dict.to_dict(), story_contents_dict)
-
-    def test_to_dict(self):
-        user_ids = [self.USER_ID, self.USER_ID_1]
-        story_rights = story_domain.StoryRights(self.STORY_ID, user_ids, False)
-        expected_dict = {
-            'story_id': self.STORY_ID,
-            'manager_names': ['user', 'user1'],
-            'story_is_published': False
-        }
-
-        self.assertEqual(expected_dict, story_rights.to_dict())
-
-    def test_is_manager(self):
-        user_ids = [self.USER_ID, self.USER_ID_1]
-        story_rights = story_domain.StoryRights(self.STORY_ID, user_ids, False)
-        self.assertTrue(story_rights.is_manager(self.USER_ID))
-        self.assertTrue(story_rights.is_manager(self.USER_ID_1))
-        self.assertFalse(story_rights.is_manager('fakeuser'))
-
-    def test_validate_non_str_exploration_id(self):
```

Comment:
Why delete these and the following? They don't seem to be story-rights related...

Reply:
Oh, yeah, accidentally deleted these. I think it might have happened when I merged with develop at some point. Reverted it.